files/sdw-notify.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=SecureDrop Updater GUI Notification
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/sdw-notify
+
+[Install]
+WantedBy=default.target
```

Comment:
https://docs.fedoraproject.org/en-US/packaging-guidelines/Systemd/#unit_files_basic_format_install says
> The recommended parameters for WantedBy= are either graphical.target (services related to the graphical user interface) or multi-user.target (for everything else).

Reply:
I didn't know this before but it looks as if the `[Install]` section for the service here we might be able to omit!

As for the timer that starts it, the `default.target` is required because neither `multi-user.target` nor `graphical.target` would apply to `systemd --user` sessions (they are already started), and if it was set to either of them or completely omitted, we wouldn't be able to enable the timer for the session.

I'll remove the `[Install]` section for the `.service` though, thanks!